experiments/tester/src/RNTester/TestComponents/ThemeTest.tsx

```diff
@@ -138,22 +157,39 @@ const ThemeTestInner: React.FunctionComponent = () => {
   const themedStyles = getThemedStyles(useTheme());
   const onAppChange = React.useCallback((app: string) => {
     brand = app;
-    customRegistry.setTheme(fakeBrandTheme, 'BrandedTheme');
+    // Invalidate the DAG children of the shimmed brand colors
+    customRegistry.setTheme(fakeBrandTheme, 'Default');
+    customRegistry.setTheme(fakeBrandTheme, 'WhiteColors', 'RealWhiteColors');
   }, []);
+
+  const [theme, setTheme] = React.useState('Default');
+  const onThemeChange = React.useCallback(
+    (theme: string) => {
```

Comment:
use can pass `setTheme` directly to `useCallback`:

`const onThemeChange = React.useCallback(setTheme, [setTheme]);`

Reply:
Whing brings out the question, why do you need this at all? You can use `setTheme` directly.